manager/manager.go

```diff
@@ -47,10 +47,14 @@ type Manager struct {
 	sender           *http.Client
 }
 
-type filterDispatcher struct {
-	norn       model.Norn
+type nornFilter struct {
+	norn   model.Norn
+	filter Filterer
+}
+
+type endpointDispatcher struct {
+	endpoint   string
 	dispatcher dispatch.D
```

Comment:
```suggestion
```
I don't think this is needed anymore, right?

Reply:
it's being used in Update()
[https://github.com/xmidt-org/mimisbrunnr/blob/3427a4e8c45ae4f23998f314647df14c4078446e/manager/manager.go#L100](url)